### DISCUSSION TITLE
No RTF values are collected

### DISCUSSION THREAD

**Discussion by sbegg** [mulesoft-catalyst/metrics-toolkit#143] [Q&A]
I just merged the latest master commit to my local. I am running it over and over, but it will not report any RTF statistics.
Is anyone able to collect RTF stats? Did you have to make any code changes?
Thank you.

**Comment by sbegg** [comment]
Update:
It may be that some metrics could be collected, but the Mule only examines the last known business group in the hierarchy, and in my case that business group has no applications or resources in any of our 3 RTF clusters. Other business groups do though.

**Comment by richardmckinley** [comment]
I have done some testing on my own account and was not able to replicate this. 

My fabric is shared with 2 business groups and metrics are retrieved for both 
BG 1 - Org level 
![image](https://user-images.githubusercontent.com/58299919/149154675-0f1dc74d-abdb-49d7-85f0-082e207f4957.png)

BG 2 - Sub-Business Group 
![image](https://user-images.githubusercontent.com/58299919/149154822-73cd9289-f0f7-4f4a-b4d0-e9214571b423.png)

I would recommend double checking that the user you are using has adequate permissions at each BG and Env. 
Also, both the RTF and ARM collectors need to be enabled at a minimum to get both fabric and app deployment metrics.

**Reply by sbegg** [reply]
Indeed the collectors do span all the BG and fabrics in my org. I checked it too. It seems that it is the Splunk dashboard reports that are not accommodating or consolidating the data correctly. Not being a Splunk query expert, I'll give up on Splunk for now.
